homeassistant/components/bluetooth/passive_update_processor.py

```diff
@@ -180,19 +259,26 @@ def __init__(
             list[Callable[[PassiveBluetoothDataUpdate[_T] | None], None]],
         ] = {}
         self.update_method = update_method
-        self.entity_names: dict[PassiveBluetoothEntityKey, str | None] = {}
-        self.entity_data: dict[PassiveBluetoothEntityKey, _T] = {}
-        self.entity_descriptions: dict[
-            PassiveBluetoothEntityKey, EntityDescription
-        ] = {}
-        self.devices: dict[str | None, DeviceInfo] = {}
+        data: PassiveBluetoothDataUpdate[_T] = PassiveBluetoothDataUpdate()
+        self.data = data  # This is the combined data from all the updates
```

Comment:
Will this conflict with active data?  Should it be passive data and only set when coordinator is set?

Reply:
Nope data is fine but should only be set when coordinator is set